src/editor/Core/BufferMap.re

```diff
@@ -13,13 +13,33 @@ module Buffers =
   });
 
 let empty = Buffers.empty;
+[@deriving show]
 type t = Buffers.t(Buffer.t);
 
 type mapFunction = Buffer.t => Buffer.t;
 
 let map = Buffers.map;
 let update = Buffers.update;
 
+let _unwrap = v =>
+  switch (v) {
+  | Some(v) => v
+  | None => ""
+  };
+
+let log = (m: t) =>
```

Comment:
this is a stop gap whilst I couldn't get `show` to work

Reply:
Actually trying to log a full `Map` is a bit involved an from what I can see very messy as it would log the entire thing including its methods :thinking: (not that I actually got it to work) I essentially used show of all the other variables to be able to log the contents here.

Let me know if you happened to get it to work differently, I still encountered the `Buffers.pp` issue having tried what you suggested and it seems it was because the `t` refers to the `Buffers` Module and trying to use `show` on that leads to the problem I described above